src/common/src/session_config/mod.rs

```diff
@@ -52,6 +52,10 @@ pub enum SessionConfigError {
 
 type SessionConfigResult<T> = std::result::Result<T, SessionConfigError>;
 
+// NOTE(kwannoel): We declare it separately as a constant,
+// otherwise seems like it can't infer the type of -1 when written inline.
```

Comment:
What about `-1i32`?

Reply:
Yeah it didn't compile.

Reply:
Only `(-1i32)` works, but then clippy complains..